core/test/VoteTiming.js

```diff
@@ -22,85 +17,13 @@ contract("VoteTiming", function(accounts) {
-
-    // If the new time is 200+ larger than startTime, the method should increment the roundId.
-    await voteTiming.wrapUpdateRoundId(initialStartTime + 5060);
-
-    // Grab the updated round id and start time.
-    const updatedRoundId = (await voteTiming.wrapGetLastUpdatedRoundId()).toNumber();
-    const updatedStartTime = (await voteTiming.voteTiming()).roundStartTime.toNumber();
-
-    // The updated round id should be incremented by 1.
-    assert.equal(updatedRoundId, initialRoundId + 1);
-
-    // The updated start time should be the provided start time floored to the nearest 200.
-    assert.equal(updatedStartTime, initialStartTime + 5000);
-
-    // If the time doesn't change, the round id and start time should not change.
-    await voteTiming.wrapUpdateRoundId(updatedStartTime);
-    assert.equal((await voteTiming.wrapGetLastUpdatedRoundId()).toNumber(), updatedRoundId);
-    assert.equal((await voteTiming.voteTiming()).roundStartTime.toNumber(), updatedStartTime);
-
-    // If the time moves forward by less than 200, the round id and time should not change.
-    await voteTiming.wrapUpdateRoundId(updatedStartTime + 199);
-    assert.equal((await voteTiming.wrapGetLastUpdatedRoundId()).toNumber(), updatedRoundId);
-    assert.equal((await voteTiming.voteTiming()).roundStartTime.toNumber(), updatedStartTime);
-
-    // If time goes backwards the round id and start time should not change.
-    await voteTiming.wrapUpdateRoundId(updatedStartTime - 1000);
-    assert.equal((await voteTiming.wrapGetLastUpdatedRoundId()).toNumber(), updatedRoundId);
-    assert.equal((await voteTiming.voteTiming()).roundStartTime.toNumber(), updatedStartTime);
-
-    // If the time moves forward by exactly 200, the round id and start time should incrmement.
-    await voteTiming.wrapUpdateRoundId(updatedStartTime + 200);
-    assert.equal((await voteTiming.wrapGetLastUpdatedRoundId()).toNumber(), updatedRoundId + 1);
-    assert.equal((await voteTiming.voteTiming()).roundStartTime.toNumber(), updatedStartTime + 200);
-  });
-
-  it("Estimated Round End Times", async function() {
-    const roundId = (await voteTiming.wrapGetLastUpdatedRoundId()).toNumber();
-    const startTime = (await voteTiming.voteTiming()).roundStartTime.toNumber();
-
-    // The estimated end of the current round should be one round length in the future.
-    assert.equal((await voteTiming.wrapComputeEstimatedRoundEndTime(roundId)).toNumber(), startTime + 200);
-
-    // The estimated end of the next round should be two round lengths in the future.
-    assert.equal((await voteTiming.wrapComputeEstimatedRoundEndTime(roundId + 1)).toNumber(), startTime + 400);
-
-    // The estimated end of the round 50 rounds from now should be 51 round lengths in the future.
-    assert.equal((await voteTiming.wrapComputeEstimatedRoundEndTime(roundId + 50)).toNumber(), startTime + 51 * 200);
+  it("Compute Round Id", async function() {
+    const startTime = Math.floor(Date.now() / 1000);
```

Comment:
Slight preference for hardcoding values in tests rather than using run-specific values.

The reason for this: if your input is run-specific, then you have to reimplement the logic that you're testing in the test. In this simple case, that logic is the `Math.floor(startTime / 200)`. If there's a logical mistake in the contract code, it's likely that you'll repeat that mistake in the test. If you were, instead, to use a hardcoded value like `11946`, you can intuitively determine that the output should be `11800` without having to implement the logic. Obviously this case is fairly simple, but I've found this principle helpful in general.